src/VTMap.js

```diff
@@ -0,0 +1,371 @@
+    guessHandler = (evt) => {
+        if (evt.target.textContent === 'Guess') {
+            this.setState({
+                modalDisplayed: true
+            })
+        }
+    }
+
+    moveHandler = (evt) => {
+        evt.preventDefault()
+
+        const {locationArray} = this.state
+        if (this.state.playing === true) {
+            let direction = evt.target.textContent
+            switch (direction) {
+                case 'North':
+
+                    
+                    let northLat = this.state.mapLat + 0.002
+                    const northLatLonArray = locationArray.concat([[northLat, this.state.mapLon]])
+                    this.setState((prevState) => {
+                        return {
+                        mapLat: northLat,
+                        points: prevState.points -= 1,
+                        locationArray: northLatLonArray,
+                        }
+                    })
+                    break
+
+                    
+                case 'South':
+
+                    // const {locationArray} = this.state
+                    let southLat = this.state.mapLat - 0.002
+                    const southLatLonArray = locationArray.concat([[southLat, this.state.mapLon]])
+                    this.setState((prevState) => {
+                        return{
+                        mapLat: southLat,
+                        points: prevState.points -= 1,
+                        locationArray: southLatLonArray,
+                        }
+                    })
+                
+                    break
+
+                case 'East':
+
+                    // const {locationArray} = this.state
+                    let eastLon = this.state.mapLon + 0.002
+                    const eastLatLonArray = locationArray.concat([[this.state.mapLat, eastLon]])
+
+                    this.setState((prevState) => {
+                        return{
+                        mapLon: eastLon,
+                        points: prevState.points -= 1,
+                        locationArray: eastLatLonArray,
+                        }
+                    })
+
+                    break
+
+                case 'West':
+
+                    // const {locationArray} = this.state
+                    let westLon = this.state.mapLon - 0.002
+                    const westLatLonArray = locationArray.concat([[this.state.mapLat, westLon]])
+
+                    this.setState((prevState) => {
+                        return {
+                        mapLon: westLon,
+                        points: prevState.points -= 1,
+                        locationArray: westLatLonArray,
+                        }
+                    })
+                    break
+                    case 'Return':
+                    this.setState((prevState) => {
+                        return{
+                        mapLat: this.state.startLat,
+                        mapLon: this.state.startLon
+                        }
+                    })
```

Comment:
You have the right idea, but `startLat` and `startLon` aren't defined anywhere else, so currently, the page breaks when trying to implement it.  When you first find a valid starting point, capture that `[lat,lon]` array in these values here and you should be good to go!